packages/ai-ide/src/browser/workspace-preferences.ts

```diff
@@ -38,6 +41,39 @@ export const WorkspacePreferencesSchema: PreferenceSchema = {
             items: {
                 type: 'string'
             }
+        },
+        [PROMPT_TEMPLATE_WORKSPACE_DIRECTORIES_PREF]: {
+            type: 'array',
+            title: nls.localize('theia/ai/promptTemplates/directories/title', 'Workspace-specific Prompt Template Directories'),
+            description: nls.localize('theia/ai/promptTemplates/directories/description',
+                'List of relative paths indicating folders in the current workspace to be scanned for WORKSPACE specific prompt templates. Templates from workspace directories \
+                have higher priority than global templates but lower priority than specific template files.'),
+            default: [],
+            items: {
+                type: 'string'
+            }
+        },
+        [PROMPT_TEMPLATE_WORKSPACE_EXTENSIONS_PREF]: {
+            type: 'array',
+            title: nls.localize('theia/ai/promptTemplates/extensions/title', 'Workspace-specific Prompt Template File Extensions'),
+            description: nls.localize('theia/ai/promptTemplates/extensions/description',
+                'List of file extensions in workspace-specific locations that are considered as prompt templates. When templates with the same ID exist in multiple locations, \
```

Comment:
The priority explanation is unrelated to the extensions preference and should rather be mentioned in the other 3 preferences.

Reply:
adapted